fcrepo-connector-file/pom.xml

```diff
@@ -21,6 +21,12 @@
       <groupId>javax.jcr</groupId>
       <artifactId>jcr</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.modeshape</groupId>
+      <artifactId>modeshape-jcr</artifactId>
+      <scope>provided</scope>
```

Comment:
Right... and ideally, most of the Fedora modules would not even depend on "pure" JCR. I would like to see as few modules as possible care anything about JCR.